### PR TITLE
[OC-1176]: ipv6 support for px-central.

### PR DESCRIPTION
**Pull request by diptiranjanpx**
Signed-Off-By: Diptiranjan

**What this PR does / why we need it**:
px-central components needs to work with pods having IPV6 address.

**Which issue(s) this PR fixes** (optional)
Closes # OC-1176

**Special notes for your reviewer**:
*Test*
```
[root@arvind-centos79 helm]# helm install px-central charts/px-central --namespace central --create-namespace --set persistentStorage.enabled=true,persistentStorage.storageClassName=px-replicated,useIPV6=true -f values-px-upgrade.yaml

[root@arvind-centos79 helm]# k get po
NAME                                       READY   STATUS      RESTARTS   AGE
pxcentral-apiserver-758749c896-sjwj2       1/1     Running     0          13m
pxcentral-backend-56c6489f59-trhrh         1/1     Running     0          9m11s
pxcentral-frontend-5fc65bbb7-gmtvw         1/1     Running     0          9m11s
pxcentral-keycloak-0                       1/1     Running     0          13m
pxcentral-keycloak-postgresql-0            1/1     Running     0          13m
pxcentral-lh-middleware-647bccf86f-2npv6   1/1     Running     0          9m11s
pxcentral-mysql-0                          1/1     Running     0          13m
pxcentral-post-install-hook-6f5tx          0/1     Completed   0          13m


[root@arvind-centos79 helm]# kubectl label node node01 px/ls=true
node/node01 labeled
[root@arvind-centos79 helm]# kubectl label node node02 px/ls=true
node/node02 labeled

[root@arvind-centos79 helm]# helm upgrade px-central charts/px-central --namespace central --create-namespace --set persistentStorage.enabled=true,persistentStorage.storageClassName=px-replicated,useIPV6=true,pxlicenseserver.enabled=true -f values-px-upgrade.yaml

[root@arvind-centos79 helm]# k get po
NAME                                        READY   STATUS      RESTARTS        AGE
pxcentral-apiserver-758749c896-sjwj2        1/1     Running     0               7d13h
pxcentral-backend-6c86f554c-x5q4g           1/1     Running     0               5m56s
pxcentral-frontend-fcc584ff9-m6qb9          1/1     Running     0               5m56s
pxcentral-keycloak-0                        1/1     Running     6 (4d5h ago)    4d6h
pxcentral-keycloak-postgresql-0             1/1     Running     14 (3d8h ago)   4d5h
pxcentral-lh-middleware-647bccf86f-nx5xl    1/1     Running     0               5m56s
pxcentral-license-server-79b66dbcc5-dx6dh   1/1     Running     0               12m
pxcentral-license-server-79b66dbcc5-tfqwm   1/1     Running     0               12m
pxcentral-mysql-0                           1/1     Running     0               27m
pxcentral-post-install-hook-bhthm           0/1     Completed   0               12m
```

### CODE REVIEW

**Review by px-kesavan — APPROVED** [review]
LGTM

File: charts/px-central/templates/px-lighthouse/px-central-oidc/pxcentral-keycloak.yaml [hunk 1]
```diff
@@ -462,6 +462,8 @@ spec:
               value: /secrets/password
             - name: JAVA_TOOL_OPTIONS
               value: "-XX:+UseContainerSupport -XX:MaxRAMPercentage=50.0"
+            - name: JAVA_OPTS
+              value: "-Xms64m -Xmx512m -XX:MetaspaceSize=96M -XX:MaxMetaspaceSize=256m -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true"
```
Comment (px-kesavan) [inline charts/px-central/templates/px-lighthouse/px-central-oidc/pxcentral-keycloak.yaml:466]: Is this related to IPV6 change ?

Comment (diptiranjanpx) [reply charts/px-central/templates/px-lighthouse/px-central-oidc/pxcentral-keycloak.yaml:466]: keycloak figures out the IPV6 but only if the the JAVA_OPTS is provided then it actually does it but otherwise it will go with default flow. So this is a way to force keycloak to figureout the IPV4 or V6 internally.

Comment (px-kesavan) [reply charts/px-central/templates/px-lighthouse/px-central-oidc/pxcentral-keycloak.yaml:466]: Thanks for the info!
